services/gateway/sources/mappings/account.js

```diff
@@ -27,6 +28,12 @@ module.exports = {
 		rewards: '=,string',
 		username: '=,string',
 		vote: '=,string',
+		totalVotesReceived: ',string',
+		isBanned: '=',
+		status: '=,string',
+		pomHeights: '=',
```

Comment:
For a consistent experience, should `pomHeights` be restructured as an array of objects containing `start` and `end` properties just like in delegates? Refer to the structure definition below:
```
	pomHeights: ['pomHeights', {
		start: '=,string',
		end: '=,string',
	}],
```

Reply:
Updated